specification/resources/resource-manager/readme.go.md

```diff
@@ -2,6 +2,16 @@
 
 These settings apply only when `--go` is specified on the command line.
 
+### Fix up regular expressions to support Unicode.
+
+``` yaml 
+directive:
+  from: swagger-document # do it globally
+  where: $.paths..pattern
```

Comment:
so is this where pattern restrictive enough ( i.e. are updating only what we want )

Reply:
This will affect all the swaggers for this config which I think is what we want but I'm still confirming.

Reply:
This `where` clause is a bit too greedy after all.  Working on narrowing it down.

Reply:
Fixed.